api/modites.js

```diff
@@ -41,7 +41,7 @@ const appendModiteProfiles = async modites => {
 
   const profileFetches = modites.map(modite => {
     return new Promise(async resolve => {
-      const profileObj = await MODITES.get(modite.id, 'json')
+      const profileObj = await modites.get(modite.id, 'json')
```

Comment:
this is a different thing. They are intentionally UPPERCASE because the values are stored in the Key-Value database on Cloudflare